src/index.d.ts

```diff
@@ -979,7 +979,13 @@ declare namespace admin.remoteConfig {
      * Publishes a Remote Config template.
      *
      * @param template The Remote Config template to be published.
-     * @param options Optional options object when publishing a Remote Config template.
+     * @param options Optional options object when publishing a Remote Config template:
+     *    - {boolean} `force` Setting this to `true` forces the Remote Config template to 
```

Comment:
Does this render as a bullet list, or just literally as a dash?  

If it's a list, I guess the idea is that the object may later contain more than just one option?

Reply:
This will render as a list and yes, we wanted to keep it open for possible future options. :)
Following is how the list would look like:
![image](https://user-images.githubusercontent.com/55609/78950409-aaf6a880-7a9c-11ea-8352-1efda6eca14f.png)

Reply:
LG, thanks!